cpp/include/Ice/Comparable.h

```diff
@@ -113,80 +114,91 @@ namespace Ice
         }
     };
 
-    //
-    // Relational operators for generated structs and classes
-    //
-
-    /**
-     * Relational operator for generated structs and classes.
-     * @param lhs The left-hand side.
-     * @param rhs The right-hand side.
-     * @return True if the left-hand side compares less than the right-hand side, false otherwise.
-     */
-    template<class C, typename = std::enable_if<std::is_member_function_pointer<decltype(&C::ice_tuple)>::value>>
-    bool operator<(const C& lhs, const C& rhs)
+    namespace Tuple
```

Comment:
Move to separate sub-namespace because it's cleaner and works around a clang bug. See comment in Proxy.cpp.